### PR TITLE
524: Add Number Formatters, 519: Add yValue to yAxisTooltipFormatter signature

### LINKED ISSUE #519: Improve yAxisTooltipFormatter
There are cases where it would be nice to have to unformatted value passed to the `yAxisTooltipFormatter`. Let's add another argument to the callback signature that contains the raw y value.

Current signature: 

```
(yField, yValueFormatted) => {}
```

Proposed signature:

```
(yField, yValueFormatted, yValue) => {}
```

This will need to be done in three components:

- src/components/BarChart/BarChart.jsx
- src/components/Bars/Bars.jsx
- src/components/LineChart/LineChart.jsx

### LINKED ISSUE #524: Add number formatter to constants
As per discussion in this ticket: https://jira.corp.appnexus.com/browse/YMP-1733

```
Example:
487 -> 487
4,872 -> 4.9K
4,872,156 -> 4.87M
4,002, 156 -> 4M
```

### PR DESCRIPTION
**Pull request by reddyesha**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [ ] IE11 (Win 7)
  - [ ] Edge (Win 10)
- [ ] Unit tests written (`common` at minimum)
- [ ] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [ ] One core team UX approval

Fixes #519
Fixes #524

### CODE REVIEW

File: src/components/LineChart/LineChart.jsx [hunk 1]
```diff
@@ -226,11 +226,16 @@ const LineChart = createClass({
...
+		 * y-value.
 		 *
-		 * Signature: `(yField, yValueFormatted) => {}`
+		 * Signature: `(yField, yValueFormatted, yValue) => {}`
 		 */
 		yAxisTooltipFormatter: func,
+		/**
+		 * An optional function used to format data in the tooltips.
+		 */
+		yAxisTooltipDataFormatter: func,
```
Comment (reddyesha) [inline src/components/LineChart/LineChart.jsx]: To allow for different formatting in tooltips. Open to ideas.

File: src/components/LineChart/LineChart.jsx [hunk 2]
```diff
@@ -255,6 +260,10 @@ const LineChart = createClass({
 		 */
 		y2AxisFormatter: func,
 		/**
+		 * An optional function used to format data in the tooltips.
+		 */
+		y2AxisTooltipDataFormatter: func,
```
Comment (reddyesha) [inline src/components/LineChart/LineChart.jsx]: To allow for different formatting in tooltips. Open to ideas.

File: src/components/LineChart/LineChart.jsx (hunk 1 above)
Comment (reddyesha) [inline src/components/LineChart/LineChart.jsx]: Would the `yAxisTooltipFormatter` signature be a better place for this? Only issue with that is, it wouldn't allow for formatting the two axes differently.

**Comment by mute** [comment]
Closing in favor of #532

**Pull request closed without merging**